_current_challenges/01_workshops.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: single
+sidebar:
+  nav: workshops.md
+title: Upcoming SAMPL workshops
+excerpt: Details of upcoming SAMPL in-person and virtual workshops, including agendas as applicable.
+permalink: /current/workshops/
+---
+
+## The SAMPL6 LogP Virtual Workshop
+
+We have a virtual pre-workshop on the SAMPL6 LogP challenge on Thursday, May 19, beginning at 7 am US Pacific time.  
+
+### Overview/Purpose
+
+The purpose of this workshop is to go over a preliminary evaluation of results, begin considering analysis and lessons learned, and nucleate opportunities for follow up and additional discussion. Part of the goal is to facilitate discussion so that participants can work *together* to maximize lessons learned in the lead up to the August in-person D3R/SAMPL workshop and the JCAMD special issue.
+We hope this early workshop will result in additional follow-up afterwards.
+
+### Agenda
+
+A tentative agenda for this workshop is (all times US Pacific):
+- **7-7:10 am**: *Welcome, introductions, and purpose*; David Mobley (UCI)
+- **7:10-7:50 am**: *Experiments and overview of results*, Mehtap Isik (MSKCC); 30 minute talk + 10 minutes of questions/discussion.
+- **7:50-8:20 am**: Christopher Loschen (Cosmologic); 20 minute talk + 10 minutes of questions/discussion
+- **8:20-8:40 am**: Nicolas Tielker (TU Dortmund); 12 minute talk + 8 minutes of questions/discussion
+- **8:40-9:00 am**: Esteban Vohringer-Martinez (Univ. Concepción); 15 minute talk + 5 minutes of questions/discussion.
+- **9:00 am-9:10 am**: Alexey Nikitin; 5 minute talk + 5 minutes of discussion.
```

Comment:
Is 5 minutes really enough for Alexey Nikitin? Shouldn't we give him at least 10 minutes?

Reply:
He wanted 5 minutes.